app/utilities/stripe_utilities/_form.html.erb

```diff
@@ -1 +1,5 @@
 <%= render "password_field", form: form, attribute: :api_token %>
+<small>
+  <%= t('.api_key_help') %>
+  <%= link_to "https://dashboard.stripe.com/apikeys", "https://dashboard.stripe.com/apikeys" %>
+</small>
```

Comment:
It would be cool if our `password_field` was able to identify if there is help text for the attribute from our locales files and emit that; but we don't have to do that now.